ConwaysGameOfLife.Tests/GameControllerShould.cs

```diff
@@ -120,9 +69,9 @@ public void DisplayUniverse_GivenValidUniverseString(string input, string expect
         {
             _input.GetReadLine(input);
             _input.GetReadLine("q");
-            
+
             _controller.Run();
-            var actual = _output.GetLastWriteLine();
+            var actual = _output.OutputList[^4];

```

Comment:
As an outsider reading your code, I don't want to have to go into your function implementation to determine why it's checking the 4th last element of your array. I suggest wrapping this in a function and giving it a clear name so the intent of this is clear. One way to do this is to add an additional function in your stub output - something like DisplayUniverse which will do this

Reply:
It varies which output is expected to be a formatted `Universe` . Not entirely sure how best to approach this. Could just give `actual` a more meaningful name e.g. `actualFormattedUniverse`.